packages/test-app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@urturn/test-app",
```

Comment:
This `package.json` file is almost verbatim to our `tictactoe` example. We test the runner by running the dev command of test-app:
`lerna run dev --scope=@urturn/test-app`

Reply:
Ill update all the related dev documentation for this.